### PR TITLE
Updated Polish translation for 3.7.0 release

### PR DESCRIPTION
**Pull request by SeeLook**

### CODE REVIEW

**Review by ann0see — APPROVED** [review]
Thanks! Unfortunately I can't comment much here. Can you test if the translation is correct by checking the artifact?

**Comment by SeeLook** [comment]
Checked and even ironed out some kinks.

File: src/res/translation/translation_pl_PL.ts [hunk 1]
```diff
@@ -952,7 +952,11 @@
     <message>
         <location filename="../../clientdlgbase.ui" line="584"/>
         <source>Set up your audio, connect to a server and start jamming!</source>
+<<<<<<< HEAD
```
Comment (pljones) [inline src/res/translation/translation_pl_PL.ts:955]: This is a failed rebase which needs resolving.

Comment (SeeLook) [reply src/res/translation/translation_pl_PL.ts]: Sorry for trouble, next shoot should be correct.

File: src/res/translation/translation_pl_PL.ts [hunk 2]
```diff
@@ -118,7 +118,7 @@
     <message>
         <location filename="../../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2021 The Jamulus Development Team</source>
-        <translation type="unfinished"></translation>
+        <translation>Copyright (C) 2005-2021 Zespół programistów Jamulus-a</translation>
```
Comment (hoffie) [inline src/res/translation/translation_pl_PL.ts:121]: We should wait with the merge depending on what we settle on in https://github.com/jamulussoftware/jamulus/pull/1097#discussion_r581265793

File: src/res/translation/translation_pl_PL.ts [hunk 3]
```diff
@@ -2973,7 +2973,7 @@
     <message>
         <location filename="../../global.h" line="237"/>
         <source>For more information use the &quot;What&apos;s This&quot; help (help menu, right mouse button or Shift+F1)</source>
-        <translation>Aby uzyskać więcej informacji użyj &quot;Co to&quot; (Pomoc, prawy przycisk myszy lub Shift+F1)</translation>
+        <translation>Aby uzyskać więcej informacji użyj „Co to jest” (z menu Pomoc lub prawy przycisk myszy lub Shift+F1)</translation>
```
Comment (hoffie) [inline src/res/translation/translation_pl_PL.ts:2976]: I assume you have verified that these quotes work properly in the UI?

**Review by hoffie — APPROVED** [review]
Thanks for your work! Looks good to me. I don't speak Polish but I have at least checked punctuation, spacing and technical words.

I wonder why there is any empty `src/res/translation/translation_pl_PL.qm` file. Not sure if it causes any problems, I guess it will be overwritten later anyway.

File: src/res/translation/translation_pl_PL.ts (hunk 3 above)
Comment (SeeLook) [reply src/res/translation/translation_pl_PL.ts:2976]: Yes. I have been using those for a long in another Qt project I'm translating and it appears correctly under any OS-es with Polish localization.

**Comment by hoffie** [comment]
Just checked: There don't seem to be any untranslated things left, which is good! :)

**Comment by SeeLook** [comment]
That was the plan :)

File: src/res/translation/translation_pl_PL.ts (hunk 1 above)
Comment (pljones) [reply src/res/translation/translation_pl_PL.ts]: Thanks!

File: src/res/translation/translation_pl_PL.ts (hunk 2 above)
Comment (ann0see) [reply src/res/translation/translation_pl_PL.ts:121]: I think we can just leave it like that. If the qm file isn't something we need to worry about I'd say this is ready to be merged then.

**Comment by pljones** [comment]
> I wonder why there is any empty `src/res/translation/translation_pl_PL.qm` file. Not sure if it causes any problems, I guess it will be overwritten later anyway.

I don't remember the process -- @softins can you comment?

**Comment by softins** [comment]
Yes, no need to worry about the .qm files. Once all the .ts files are in and merged, one of us (I'm happy to) can just run `lrelease Jamulus.pro`, which will generate all the .qm files and then check them in. I think really they ought to be generated by the makefile, but that doesn't happen now. I tried deleting the qms and runnng make, and it failed, complaining it couldn't find them.

**Pull request merged**